src/plugins/data/public/ui/dataset_selector/configurator.test.tsx

```diff
@@ -0,0 +1,182 @@
+/*
```

Comment:
Are the lines called out as missed by codecov in configurator.tsx not considered critical?